### ISSUE TITLE
DEPRECATION: An addon is trying to access project.nodeModulesPath. 

### ISSUE THREAD

**Issue by broerse** [alexspeller/ember-cli-active-link-wrapper#26]
I get this deprecation in node v8.9.4. Not sure if it is this addon. Perhaps update `ember-cli-dependency-checker` to : "^2.1.0" ??

```
DEPRECATION: An addon is trying to access project.nodeModulesPath. This is not a reliable way to discover npm modules. Instead, consider doing: require("resolve").sync(something, { basedir: project.root }). Accessed from:   new NPMDependencyVersionChecker (C:\dev\ember-cli-blog\node_modules\ember-cli-active-link-wrapper\node_modules\ember-cli-version-checker\src\npm-dependency-version-checker.js:11:33)
````

**Comment by nmcclay** [comment]
I am also seeing this error after updating to Ember 2.18.0

**Comment by broerse** [comment]
I think the deprication comes from `"ember-cli-babel": "^5.1.5"` so perhaps we just need to update that?

**Comment by bungeye** [comment]
ember-cli-babel ^5.1.5 installs ember-cli-version-checker@1.3.1 which causes deprecation warning as per:
[https://github.com/ember-cli/ember-cli-version-checker/issues/48#issuecomment-354912902](https://github.com/ember-cli/ember-cli-version-checker/issues/48#issuecomment-354912902)
so updating ember-cli-babel should/will fix it

**Comment by albertosouza** [comment]
@alexspeller can you take a look here? 
I can help with some support in this module if you need

**Comment by alexspeller** [comment]
@albertosouza I have tried the upgrade but this causes the tests to fail. Are you able to make a PR to fix?

**Comment by albertosouza** [comment]
@alexspeller I'll try to fix it until friday.

**Comment by jevanlingen** [comment]
Is this issue still something being worked on?

**Comment by albertosouza** [comment]
@jevanlingen sorry but i had no time for this :(

**Comment by broerse** [comment]
@albertosouza I hope you can find some time soon.

**Comment by alexspeller** [comment]
Fixed with v0.5.0
